PREP_SYSTEM_FOR_DIETPI.sh

```diff
@@ -413,98 +440,62 @@ _EOF_
 	#------------------------------------------------------------------------------------------------
 
 	# - DietPi list of minimal required packages which must be installed:
-	#	dpkg --get-selections | awk '{print $1}' | sed 's/:armhf//g' | sed "s/^/'/g" | sed "s/$/'/g"
+	#	dpkg --get-selections | awk '{print $1}' | sed 's/:a....//g' | sed "s/^/'/g" | sed "s/$/'/g"
```

Comment:
@MichaIng 
Nice 👍

Reply:
Hehe, does not work on all architectures, but at least armhf, armel, arm64, amd64 😆.